newswires/client/eslint.config.mjs

```diff
@@ -0,0 +1,25 @@
+import guardian from '@guardian/eslint-config';
+import storybook from 'eslint-plugin-storybook';
+import newswiresConfig from '../../eslint.config.mjs';
+
+export default [
+	...newswiresConfig,
+	...guardian.configs.react,
+	...storybook.configs['flat/recommended'],
+	{
+		rules: {
+      "@typescript-eslint/switch-exhaustiveness-check": [
+        "error",
+        {considerDefaultExhaustiveForUnions: true},
```

Comment:
I had to add this in as there's a few bits of code that fall foul of the default for this rule, which does not consider a `default` case as exhausting all the options for the switch. We could consider coming back and refactoring to not use `defaults` and turning this option off...

Reply:
seems v sensible to me 👍